.travis.yml

```diff
@@ -49,8 +52,8 @@ matrix:
         - env: DJANGO_VERSION=">=1.9,<1.10"
         - python: "pypy"
 
-services:
-    - elasticsearch
+#services:
```

Comment:
using a build matrix seems the best solution, maybe with only elasticsearch-1.7 and elasticsearch-2.1 versions